cumulusci/core/source_transforms/transforms.py

```diff
@@ -163,15 +164,17 @@ class RemoveFeatureParametersTransform(SourceTransform):
 
     identifier = "remove_feature_parameters"
 
-    def process(self, zf: ZipFile, logger: Logger) -> ZipFile:
+    def process(self, zf: ZipFile, context: TaskContext) -> ZipFile:
         package_xml = None
         zip_dest = ZipFile(io.BytesIO(), "w", zipfile.ZIP_DEFLATED)
```

Comment:
@TheBitShepherd Just for my learning, what if the zip content is more than 5GB, what will happen in this case ? Curious to know the upper limit of zip file

Reply:
@kdvenkata - The [MetaData API](https://developer.salesforce.com/docs/atlas.en-us.salesforce_app_limits_cheatsheet.meta/salesforce_app_limits_cheatsheet/salesforce_app_limits_platform_metadata.htm) currently only supports deploy/retrieve operations with z ipfiles that are less than 39MB. Docs: